development/dev-image/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM nutsfoundation/nuts-node:latest as nutsnode
```

Comment:
I'd say from `master`

Reply:
`make docker` uses tag `latest`. So building the dev-image locally would then require an extra step. I'd prefer to change both to `master`